src/GitHub.Api/Application/ApiClient.cs

```diff
@@ -198,22 +210,55 @@ public ApiClient(UriString hostUrl, IKeychain keychain, IGitHubClient githubClie
                 await ValidateKeychain();
                 await ValidateCurrentUserInternal();
 
-                GitHubRepository repository;
+                var uriString = keychain.Connections.First().Host;
+                var keychainAdapter = await keychain.Load(uriString);
+
+                var command = new StringBuilder("publish -r ");
+                command.Append(newRepository.Name);
```

Comment:
This should be surrounded with quotes. Not that there will be spaces in this name, but there might be other characters that could be interpreted by the shell and cause problems.

Reply:
👍